pom.xml

```
@@ -38,33 +37,40 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>junit</artifactId>
-      <version>1159.v0b_396e1e07dd</version>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
-      <version>2.6.1.1</version>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plain-credentials</artifactId>
-      <version>1.5
-      </version>
     </dependency>
 
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.7</version>
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+      <dependencies>
+          <dependency>
+              <groupId>io.jenkins.tools.bom</groupId>
+              <artifactId>bom-2.361.x</artifactId>
+              <version>1887.vda_d0ddb_c15c4</version>

```

Most recent version is 1981.v17df70e84a_a_1
```suggestion
              <version>1981.v17df70e84a_a_1</version>
```